### ISSUE TITLE
False positive on lombok @EqualsAndHashCode and generated methods

### ISSUE THREAD

**Issue by willypm33** [spotbugs/sonar-findbugs#1255]
SonarQube Developer Edition v2025.2 (105476)
sonar-findbugs v4.4.2

For **EqualsAndHashCode** annotation, we have the rule ID findbugs:RC_REF_COMPARISON on **Generated** method **hashCode** by Lombok.

**Comment by gtoison** [comment]
I think this should be fixed in your lombok config with:
```
lombok.addLombokGeneratedAnnotation = true
lombok.extern.findbugs.addSuppressFBWarnings = true
lombok.addSuppressWarnings = false
```

See: https://projectlombok.org/features/configuration
